docs/design/expressions/implicit_conversions.md

```diff
@@ -101,27 +101,30 @@ salient part of a `StringView`'s value.
 
 The following implicit numeric conversions are available:
 
--   `iN` or `uN` -> `iM` if `M` > `N`
--   `uN` -> `uM` if `M` > `N`
```

Comment:
I find it easier to read this using the short type names. Is there a motivation to documenting with the long ones here?

Reply:
Do we allow the short type names for arbitrary `N` or just powers of 2?

Reply:
I somewhat hope they have the same restrictions, whatever those end up being.

I think the most problematic one for the short syntax will have to be covered either way: `i1`.

Reply:
I'm going to put this PR on hold until #2015 is resolved.

Reply:
I've switched this to follow the expected resolution from #2015

Reply:
#2015 has been merged so I'm resolving this.